Apps/Src/ReadCarCan.c

```diff
@@ -368,15 +368,17 @@ void TaskReadCarCan(void *p_arg) {
     memset(hv_plus_minus_charge_msg_buffer, DISABLE_SATURATION_MSG,
            sizeof(hv_plus_minus_charge_msg_buffer));
 
-    while (1) {
+    LOOP {
         UpdatePrechargeContactors();  // Sets array and motor controller PBC if
                                       // all conditions (PBC Status, Threshold,
                                       // Precharge Complete) permit
 
         // BPS sent a message
         ErrorStatus status = CanBusRead(&data_buf, true, CARCAN);
         if (status != SUCCESS) {
+#ifndef MOCKING
             continue;
+#endif
```

Comment:
MOCKING is the general macro that we define if we're unit testing, but I can't remember why it surrounds the "continue" here. My best guess is that it was added while we were still developing and then didn't get removed, so I just went ahead and deleted it since that makes the most sense to me.